SPECS-EXTENDED/perl-Params-Validate/perl-Params-Validate.spec

```diff
@@ -1,22 +1,20 @@
 Summary:        Params-Validate Perl module
 Name:           perl-Params-Validate
-Version:        1.29
-Release:        13%{?dist}
-# One file is GPL+ or Artistics (c/ppport.h)
-License:        Artistic 2.0 and (GPL+ or Artistic)
+Version:        1.31
+Release:        1%{?dist}
+# One file is GPL-1.0-or-later OR Artistic-1.0-Perl (c/ppport.h)
```

Comment:
Can you please confirm that you checked all the source files under the source tar and verified the License.